src/lib/button/_button-base.scss

```diff
@@ -86,10 +85,6 @@ $mat-mini-fab-padding: 8px !default;
   transition: background $swift-ease-out-duration $swift-ease-out-timing-function,
               mat-elevation-transition-property-value();
 
-  &:not([disabled]):active {
-    @include mat-overridable-elevation(8);
-  }
-
   &[disabled] {
     box-shadow: none;
```

Comment:
I think this needs to move to the theme too, I'm seeing screenshots with disabled buttons that have shadows (this is basically the equivalent of `mat-elevation(0)`)

Reply:
card has a similar one that should go into the theme file:
```scss
&.mat-card-flat {
  box-shadow: none;
}
```